include/swift/Driver/Job.h

```diff
@@ -271,6 +271,14 @@ class Job {
   /// Whether the job wants a list of input or output files created.
   std::vector<FilelistInfo> FilelistFileInfos;
 
+  /// Response file path
+  const char *ResponseFilePath;
+
+  /// Argument vector containing a single argument pointing to the response file
+  /// path with the '@' prefix.
+  /// The argument string must be kept alive as long as the Job is alive.
+  llvm::opt::ArgStringList ResponseFileArg;
```

Comment:
How about grouping `ResponseFilePath` and `ResponseFileArg` into a substructure? I think the extra structure makes the Job class more readable.

Reply:
The new struct or class could have a nice comment describing it's purpose and use.

Reply:
Does it make sense for the instance variable in Job holding the new structure to be optional?